src/tests/cucumber/features/steps/output_utils.py

```diff
@@ -0,0 +1,40 @@
+# Antares outputs parsing
+
+import os
+import time
+import datetime
+import pandas
+
+def parse_output_folder_from_logs(logs: bytes) -> str:
+    for line in logs.splitlines():
+        if b'Output folder : ' in line:
+            return line.split(b'Output folder : ')[1].decode('ascii')
+    raise LookupError("Could not parse output folder in output logs")
+
+
+def parse_annual_system_cost(output_path: str) -> dict:
+    file = open(os.path.join(output_path, "annualSystemCost.txt"), 'r')
+    keys = ["EXP", "STD", "MIN", "MAX"]
+    annual_system_cost = {}
+    for line in file.readlines():
+        for key in keys:
+            if key in line:
+                annual_system_cost[key] = float(line.split(key + " : ")[1])
+    return annual_system_cost
+
+
+def parse_simu_time(output_path: str) -> float:
+    file = open(os.path.join(output_path, "simulation.log"), 'r')
```

Comment:
ther is maybe straightforward way to check some data, like in output/ info.antares-output for execution time (not sure)

Reply:
or in execution_info.ini